src/snapred/backend/data/Indexer.py

```diff
@@ -145,7 +142,7 @@ def defaultVersion(self) -> int:
         """
         The version number to use for default states.
         """
-        return VERSION_DEFAULT
+        return VERSION_START
```

Comment:
Why is the default version beginning at the start?

Reply:
The internal vs external representation.  Though this indexer doesnt actually separate them totally.  
The concept of "default" probably shouldnt have bled into indexer, as that is more related to the external ways of using the indexer.  I.e. a Calibration has a default, which can be established using the Indexer, but Normalization does not.  In both cases the indexer should start at some arbitrary version and increment.

We could further flesh out the concept of "friendly names" in the future to support any special versions (an additional mapping field on the index?)